src/pages/Join.jsx

```diff
@@ -62,13 +62,16 @@ export default function Join() {
         <br></br>
 
         <div className="big-black-title">What we're looking for</div>
-        <div class="grid-container1">
-          <div class="flip-card">
-            <div class="flip-card-inner">
-              <div class="flip-card-front">
+        <div
+          className="grid-container1"
+          style={{ display: "flex", flexWrap: "wrap" }}
+        >
+          <div className="flip-card">
+            <div className="flip-card-inner">
+              <div className="flip-card-front">
                 <div className="pathway-title1">Commitment</div>
```

Comment:
For these for tiles, it is very likely we might want to add more at some point. Can you refactor the code to render the four tiles from an array? It would be even better if you made the tiles into a component. Also, since it's not that much information, it is fine if you store the descriptions inside Join.jsx. You don't have to make a new data file.